### PR TITLE
Labs rewrite

### PR DESCRIPTION
**Pull request by mike-plummer**
Sorry for the size of this - basically refactoring large sections of the labs to modernize and tighten them up a bit. Looking to use this @ GVL next week

Major changes:
1. Redid Lab02 to change from writing a HelloWorld component to implementing a segment of the overall Timesheetz app
2. Redid Lab05 to use Formik instead of React Bootstrap forms - seriously reduces boilerplate and seems a bit more useful in the scheme of things.
3. Removed manual binds and replaced with arrow functions throughout
4. Combining create & detail views to reduce boilerplate
5. General dependency updates

Diff view is pretty worthless, probably easier to just take a look at how it looks in my fork: https://github.com/mike-plummer/react-redux-workshop

### CODE REVIEW

File: final/src/actions/EmployeeActionCreator.js [hunk 1]
```diff
@@ -6,26 +6,12 @@ const apiUrl = '/api/users';
 const url = employeeId => {
   let url = apiUrl;
   if (employeeId) {
-    url += `/${employeeId}`;
+    url += '/' + employeeId;
```
Comment (JakePartusch) [inline final/src/actions/EmployeeActionCreator.js]: return `${url}/${employeeId`;  😎

**Comment by mike-plummer** [comment]
Not sure why Netlify thinks it hasn't deployed yet - it has: https://app.netlify.com/sites/react-redux-timesheet/deploys/5c2d4140a7c594000720f1bb

File: final/src/actions/EmployeeActionCreator.js (hunk 1 above)
Comment (mike-plummer) [reply final/src/actions/EmployeeActionCreator.js]: Ugh, the shame...

File: final/src/employees/EmployeeForm.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    return errors;
+  };
+
+  render() {
+    const { employee } = this.props;
+
+    return (
+      <Formik
```
Comment (JakePartusch) [inline final/src/employees/EmployeeForm.js]: 🎉

**Pull request merged**